metadata/add_plot_boundaries.Rmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "Adding plot boundaries to BETYdb"
+author: "Kimberly Huynh"
+date: "5/20/2019"
+output: html_document
+---
+
+# Objective
+
+The objective of this tutorial is to demonstrate how to add plot boundaries to the BETYdb sites table. For this tutorial, data will be added to a local instance of a test database. Instructions on how to set up a test database can be found [here](https://gist.github.com/kimberlyh66/935106e35478829b9e844ba5b496784d).
+
+Shapefiles for MAC Season 8 sites will be used as an example.
+
+# Set up
+
+Load in the `sf` and `RPostgreSQL` packages.
+
+```{r load-pack, message = FALSE}
+
+library(sf)
+library(RPostgreSQL)
+
+```
+
+Create a connection to the BETY test database.
+
+```{r create-dbcon}
+
+dbcon <- dbConnect(PostgreSQL(),
+                   host = 'localhost',
+                   user = 'bety',
+                   port = 5433)
+
+```
+
+# Adding plot boundaries using `sf` package
+
+## Read in shapefiles using `st_read`
+
+MAC Season 8 shapefiles can be read in as a `simple features object` using `st_read`. These shapefiles are saved in the `MAC_season8` folder of this directory. 
+
+A `sf` object is a dataframe where rows are features (points, lines, polygons) and columns are attributes of the features.
+
+```{r read-shape, results = 'hide'}
+
+s8_bounds <- st_read('MAC_season8/S8_one_row_polys.shp')
+
+```
+
+If you need to convert the coordinate reference system, you can use the `st_transform` function. In this example, we will be converting the EPSG from `32612` to `4326`.
+
+```{r transform-crs}
+
+s8_bounds <- st_transform(s8_bounds, 4326)
+
+```
+
+## Write a table to BETYdb using `st_write`
+
+The `st_write` function can be used to write a new table to BETYdb containing the contents of the `sf` object. The new table will be named `s8_bounds` and will be dropped after it is used to insert new data into the `sites` table.
```

Comment:
If this is not a step that's required only due to st_transformation, please add a comment along the lines of "Regardless of whether or not the shapes were transformed ..."

Reply:
Okay. This was a step that was needed regardless whether st_transformation was run.